search.json

```diff
@@ -0,0 +1,10 @@
+{
+  "GATEWAY_URL": "http://unstable--dummy-api.libero.pub",
```

Comment:
if this is a JSON object of configuration values (they don't seem environment variables), then why the keys need to be UPPERCASE? It's a Flask convention, would be my guess

Reply:
yes, It turns out any key that are not uppercase Flask will actually ignore them....